src/v2/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -1,14 +1,190 @@
-import { Box, Text } from "@artsy/palette"
+import { GridColumns, Column, FullBleed, Separator, Box } from "@artsy/palette"
+import {
+  createPaginationContainer,
+  graphql,
+  RelayPaginationProp,
+} from "react-relay"
+import { Fragment, useState } from "react"
+import { SavedSearchAlertsApp_me } from "v2/__generated__/SavedSearchAlertsApp_me.graphql"
+import { Media } from "v2/Utils/Responsive"
+import { EditAlertEntity } from "./types"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { SavedSearchAlertDeleteModal } from "./Components/SavedSearchAlertDeleteModal"
+import {
+  SavedSearchAlertListItemFragmentContainer,
+  SavedSearchAlertListItemVariant,
+} from "./Components/SavedSearchAlertListItem"
+import { SavedSearchAlertHeader } from "./Components/SavedSearchAlertHeader"
 import { MetaTags } from "v2/Components/MetaTags"
+import { SavedSearchAlertsEmptyResults } from "./Components/SavedSearchAlertsEmptyResults"
+import { SavedSearchAlertEditFormContainer } from "./Components/SavedSearchAlertEditFormContainter"
+
+interface SavedSearchAlertsAppProps {
+  me: SavedSearchAlertsApp_me
+  relay: RelayPaginationProp
+}
+
+export const SavedSearchAlertsApp: React.FC<SavedSearchAlertsAppProps> = ({
+  me,
+  relay,
+}) => {
+  const [
+    editAlertEntity,
+    setEditAlertEntity,
+  ] = useState<EditAlertEntity | null>(null)
+  const [showDeleteModal, setShowDeleteModal] = useState(false)
+  const alerts = extractNodes(me.savedSearchesConnection)
+  const isEditMode = editAlertEntity !== null
+
+  const handleCloseClick = () => {
+    setEditAlertEntity(null)
+  }
+
+  const handleCompleted = () => {
+    handleCloseClick()
+    relay.refetchConnection(50)
+  }
+
+  const handleDeleteClick = () => {
+    setShowDeleteModal(true)
+  }
+
+  const handleCloseDeleteModal = () => {
+    setShowDeleteModal(false)
+  }
+
+  const handleDeleted = () => {
+    handleCompleted()
+    handleCloseDeleteModal()
+  }
+
+  const list = (
+    <>
+      {alerts.map((edge, index) => {
+        const isLastEdge = alerts.length === index + 1
+        const isCurrentEdgeSelected = editAlertEntity?.id === edge.internalID
+        let variant: SavedSearchAlertListItemVariant | undefined
+
+        if (editAlertEntity?.id === edge.internalID) {
+          variant = "active"
+        } else if (!!editAlertEntity && !isCurrentEdgeSelected) {
+          variant = "inactive"
+        }
+
+        return (
+          <Fragment key={edge.internalID}>
+            <SavedSearchAlertListItemFragmentContainer
+              item={edge}
+              variant={variant}
+              onEditAlertClick={setEditAlertEntity}
+            />
+            {!isLastEdge && <Separator color="black15" />}
```

Comment:
**suggestion** We have a Palette component that helps with lists and separators that might work well here: https://palette.artsy.net/elements/join/

Reply:
Ooo, really good suggestion 💯